rgw/v2/tests/s3_swift/test_bucket_lifecycle_object_expiration.py

```diff
@@ -307,6 +303,12 @@ def test_exec(config, ssh_con):
                 else:
                     lc_ops.validate_prefix_rule(bucket, config)
 
+        if not config.rgw_enable_lc_threads:
+            ceph_conf.set_to_ceph_conf(
+                "global", ConfigOpts.rgw_enable_lc_threads, "true", ssh_con
```

Comment:
@anrao19 , why do we need **ssh_con** parameter?

Reply:
wherever set_to_ceph_conf method has been used, we are currently passing ssh_con parameter. This has been introduced recently